go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -99,20 +99,22 @@ func (sparkResourceHandler) BuildResource(ctx context.Context, taskCtx pluginsCo
 	}
 	driverSpec := sparkOp.DriverSpec{
 		SparkPodSpec: sparkOp.SparkPodSpec{
-			Annotations: annotations,
-			Labels:      labels,
-			EnvVars:     sparkEnvVars,
-			Image:       &container.Image,
+			Annotations:      annotations,
+			Labels:           labels,
+			EnvVars:          sparkEnvVars,
+			Image:            &container.Image,
+			SecurityContenxt: config.GetK8sPluginConfig().DefaultPodSecurityContext.DeepCopy(),
```

Comment:
Let's use the corresponding `podSecurityContext` and `securityContext` in SparkApplication based on the conf: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md#using-security-context